recipes-devtools/triton/triton-common_2.58.0.bb

```diff
@@ -6,6 +6,7 @@ SECTION = "libs"
 SRC_URI = "\
     git://github.com/triton-inference-server/common.git;protocol=https;branch=r25.05 \
     file://0001-Build-fixups.patch \
+    file://0002-cmake-Set-minimum-required-version-to-3.5-for-CMake-.patch \
```

Comment:
backporting - https://github.com/triton-inference-server/common/commit/2e41435a59f7fe1f5f73df5355ae7433a15a4650 would be nicer.